lib/src/json_schema_form.dart

```diff
@@ -1,30 +1,61 @@
+import 'dart:convert';
+
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_json_schema/src/checkbox_form_field.dart';
-import 'package:flutter_json_schema/src/json_schema_bloc.dart';
 import 'package:flutter_json_schema/src/models.dart';
 
 class JsonSchemaForm extends StatefulWidget {
-  final Schema schema;
-  final JsonSchemaBloc jsonSchemaBloc;
+  final String jsonSchema;
+  final String uiSchema;
+  final Function(String) onFormDataChanged;
```

Comment:
`final void Function(String)` would make the return type of this function explicit instead of dynamic as it is right now.